mlir/include/mlir/Dialect/Affine/Passes.td

```diff
@@ -48,6 +48,13 @@ def AffineLoopInvariantCodeMotion
   let constructor = "mlir::createAffineLoopInvariantCodeMotionPass()";
 }
 
+def AffineLoopInterchange
+    : FunctionPass<"affine-loop-interchange">{
+  let summary = "Performs loop interchange of all the affine loopnests to 
```

Comment:
of all the affine loopnests -> on all affine loop nests

Reply:
Fixed. Thanks.